### PR TITLE
Add (Issue is closed) to the 'View on Github' button if is_issue_closed.

### PR DESCRIPTION
**Pull request by pinkiebell**
- Do not update bounty title and description if already populated. Relevant #1316
- Update the bounty refresh logic.
- Derive 'github_comments'(count) from the github issue instead of fetching comments separately and counting them.
- Derive 'last_comment_date' from the last update of the github issue.

Relevant: #749

### CODE REVIEW

File: app/dashboard/models.py [hunk 1]
```diff
@@ -650,6 +651,39 @@ def get_github_api_url(self):
...
+                if item:
+                    last_update = datetime.strptime(item, '%Y-%m-%dT%H:%M:%SZ').replace(tzinfo=pytz.utc)
+                    self.last_comment_date = last_update
+                item = items.get('state', '')
+                if item:
+                    self.github_status = item
+
+                self.save()
+                return True
+        return False
```
Comment (owocki) [inline app/dashboard/models.py]: would be good to DRY with the method below this, `fetch_issue_item`

**Comment by mbeacom** [comment]
We already have access to the github issue status in the meta from GH and its updated everytime we pull updates. 🤔

**Comment by owocki** [comment]
yes

**Comment by pinkiebell** [comment]
@mbeacom @owocki 
Refactored and ready for review.

**Comment by pinkiebell** [comment]
Hmpf. Just saw that. 🙃 
Conflict #1942 and #1316

File: app/assets/v2/js/pages/bounty_details.js [hunk 2]
```diff
@@ -769,7 +769,8 @@ var do_actions = function(result) {
     const _entry = {
       enabled: true,
       href: github_url,
-      text: gettext('View On Github'),
+      text: gettext('View On Github') +
```
Comment (owocki) [inline app/assets/v2/js/pages/bounty_details.js]: is this part of the spec?

Comment (pinkiebell) [reply app/assets/v2/js/pages/bounty_details.js]: Well, the spec has a question mark at the end ^^, so I thought the best experience is to show additional information only if the github issue is closed. Open for feedback 👍

**Comment by thelostone-mc** [comment]
> Do not update bounty title and description if already populated.

closing this out -> cause we don't allow folks to edit the fund form themselves

```
Derive 'github_comments'(count) from the github issue instead of fetching comments separately and counting them.
Derive 'last_comment_date' from the last update of the github issue.
```

If you'd up for just pushing this in as a new PR -> I'd be happy to review and get it in

**Pull request closed without merging**